manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/application.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: "{{ .Release.Name }}"
+  annotations:
+    kubernetes-engine.cloud.google.com/icon: >-
+      data:image/png;base64,{{ .Files.Get "logo.png" | b64enc }}
+    marketplace.cloud.google.com/deploy-info: '{"partner_id": "click-to-deploy-containers", "product_id": "kubeflow-pipelines", "partner_name": "Google Click to Deploy"}'
+  labels:
+    app.kubernetes.io/name: "{{ .Release.Name }}"
+spec:
+  descriptor:
+    type: KUBEFLOW
+    version: '0.1.23'
+    description: |-
+      Machine Learning Pipeline on Kubernetes
+    maintainers:
+      - name: Google Click to Deploy
```

Comment:
Unless the final application will end up in https://github.com/GoogleCloudPlatform/click-to-deploy, can you list Kubeflow-related maintainers info here? (also in the README)